index.d.ts

```diff
@@ -308,20 +300,20 @@ declare namespace OAuth2Server {
         saveAuthorizationCode(
           code: Pick<AuthorizationCode, 'authorizationCode' | 'expiresAt' | 'redirectUri' | 'scope' | 'codeChallenge' | 'codeChallengeMethod'>,
```

Comment:
I accidentally found that `'codeChallenge' | 'codeChallengeMethod'` are not part of `code` but own arguments, see: https://github.com/node-oauth/node-oauth2-server/blob/development/lib/handlers/authorize-handler.js#L101

Can you please update this, too?  :-)

Reply:
Please wait with this change, I open an issue, it seems to be a code smell in the implementation!

Reply:
Nevermind, this is all correct I completely misread things :see_no_evil: